spec/calendar.html

```diff
@@ -843,9 +876,26 @@ <h1>Temporal.Calendar.prototype.fields ( _fields_ )</h1>
         1. Let _fieldNames_ be ? CreateListFromArrayLike(_fields_, « String »).
         1. Return ? CreateArrayFromList(_fieldNames_).
       </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.calendar.prototype.mergefields">
+      <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1>
+      <p>
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the `mergeFields` method as specified in the ECMA-402 specification.
+        If an ECMAScript implementation does not include the ECMA-402 API the following specification of the `mergeFields` method is used.
+      </p>
       <p>
-        This function is the <dfn>%Temporal.Calendar.prototype.fields%</dfn> intrinsic object.
+        The `mergeFields` method takes two arguments, _fields_ and _additionalFields_.
+        The following steps are taken:
       </p>
+      <emu-alg>
+        1. Let _calendar_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
+        1. Assert: _calendar_.[[Identifier]] is *"iso8601"*.
+        1. Set _fields_ to ? ToObject(_fields_).
+        1. Set _additionalFields_ to ? ToObject(_additionalFields_).
```

Comment:
if we call ToTemporal____Fields on the bags, we can change these to `Assert`s

Reply:
I'm not sure I agree with that, because user code can call `calendar.mergeFields(Math.PI, false)` regardless of what Temporal does.